pkg/daemon/ceph/osd/encryption.go

```diff
@@ -172,6 +173,17 @@ func openEncryptedDevice(context *clusterd.Context, disk, target, passphrase str
 	return nil
 }
 
+func removeEncryptedDevice(context *clusterd.Context, target string) error {
+	args := []string{"remove", "--force", target}
+	output, err := context.Executor.ExecuteCommandWithTimeout(removeEncryptedDeviceCmdTimeOut, "dmsetup", args...)
+	// ignore error if no device was found.
+	if err != nil && !strings.Contains(output, "No such device") {
```

Comment:
```suggestion
	if err != nil &&  code, ok := exec.ExitStatus(err); ok && code != int(syscall.ENODEV) {
```

Reply:
This does not work, 
The exit status here is 1 
```
output: device-mapper: remove ioctl on ocs-deviceset-thin-csi-1-data-0cg76k-block-dmcrypt  failed: No such device or address
error: Command failed.exit status 1
```

Reply:
ok fine, I think we could benefit from a Debug log for both success and silenced failure.